lib/src/screen_lock.dart

```diff
@@ -102,7 +101,7 @@ class ScreenLock extends StatefulWidget {
   final InputButtonConfig inputButtonConfig;
```

Comment:
it's courtesy to normally also pass `BuildContext` as the first parameter to any widget builder. I forgot about it.